test-skripsi/test-improvement.tsx

```diff
@@ -38,7 +52,7 @@
 
 console.log(dataManager.getAllData());
 dataManager.sortData((a, b) => a - b);
-console.log(dataData.getAllData());
+console.log(dataManager.getAllData());
```

Comment:
Good catch on fixing the typo in the `console.log` call. Such attention to detail improves code readability and debuggability.